src/types/api/submission.ts

```diff
@@ -6,3 +8,21 @@ export type SubmissionResponseDto = {
 }
 
 export type SubmissionErrorDto = ErrorDto & { spcpSubmissionFailure?: true }
+
+export type SubmissionCountQueryDto = {
+  formId: string
+  startDate?: Date
```

Comment:
should update the type to require both startDate and endDate or none of them

Reply:
thanks for pointing this out and suggesting using the XOR type! i'll refactor so it uses XOR!

Reply:
how would you use the XOR type here? XOR is "either one or the other, not both and not none", whereas here you need "either both or none, not one or the other"

this would make more sense to me:
```
{
  formId: string
  dates?: {
    startDate: Date
    endDate: Date
  }
}
```